pkg/volume/scaleio/sio_plugin.go

```diff
@@ -27,9 +27,10 @@ import (
 )
 
 const (
-	sioName           = "scaleio"
-	sioPluginName     = "kubernetes.io/scaleio"
-	sioConfigFileName = "sioconf.dat"
+	sioName            = "scaleio"
+	sioPluginName      = "kubernetes.io/scaleio"
+	sioConfigFileName  = "sioconf.dat"
+	sioSecretNamespace = "scaleio"
```

Comment:
the same comment as https://github.com/kubernetes/kubernetes/pull/53622#issuecomment-335356838 applies here. this change has a few problems:
1. it is not backwards compatible (it breaks existing volumes that use secrets in the pod namespace)
2. it hardcodes a namespace which cannot be assumed to be available

the approach outlined in https://github.com/kubernetes/kubernetes/pull/53622#issuecomment-335356838 is what should be followed to enable detaching mount credentials from the pod namespace.